packages/storefront-lwc/src/modules/commerce/errorPage/errorPage.html

```diff
@@ -0,0 +1,85 @@
+<!--
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+-->
+<template>
+    <commerce-context-heading if:true={pagenotfound} header-text="Page not found"></commerce-context-heading>
+    <commerce-context-heading if:true={servererror} header-text="Something went wrong"></commerce-context-heading>
+    <div class="container" if:true={hasError}>
+        <div if:true={devMode}>
+            <div if:true={error} class="row pt-2">
+                <pre>
+                    <code>
+                        <template for:each={error} for:item="errorLine">
+                            <span key={errorLine.index}>{errorLine.line}</span>
+                        </template>
+                    </code>
+                </pre>
+            </div>
+            <div if:true={stack} class="row pt-2">
+                <pre>
+                    <code>
+                        <template for:each={stack} for:item="stackLine">
+                            <span key={stackLine.index}>{stackLine.line}</span>
+                        </template>
+                    </code>
+                </pre>
+            </div>
+            <div if:false={error} class="row pt-4">
+                <div class="col"></div>
+                <div class="col-12 col-sm-6" if:true={pagenotfound}>
+                    <p class="text-left">We can't find the page you're looking for!</p>
+                    <p class="text-left">You can either visit home page or contact our support team.</p>
+                </div>
+                <div class="col-12 col-sm-6" if:true={servererror}>
+                    <p class="text-left">Sorry, something went wrong. Please try again or contact us.</p>
+                </div>
+                <div class="col"></div>
+            </div>
+            <div class="row pt-2">
+                <div class="col"></div>
+                <div class="col-12 col-sm-6 text-center">
+                    <p>
+                        <lwce-link href="/" class="btn btn-primary btn-block"> Homepage </lwce-link>
+                    </p>
+                </div>
+                <div class="col"></div>
+            </div>
+        </div>
+        <div if:false={devMode} class="row pt-4">
+            <div class="col"></div>
+            <div class="col-12 col-sm-6" if:true={pagenotfound}>
+                <p class="text-left">We can't find the page you're looking for!</p>
+                <p class="text-left">You can either visit home page or contact our support team.</p>
+            </div>
+            <div class="col-12 col-sm-6" if:true={servererror}>
+                <p class="text-left">Sorry, something went wrong. You can either visit home page or contact our support team.</p>
+            </div>
+            <div class="col"></div>
+        </div>
```

Comment:
L31-L39 looks to be the same as L52-L60 maybe we should consider abstracting this out into a reusable piece

Reply:
It is intentionally done to keep the devMode and prodMode separate.